documentation/optee_design.md

```diff
@@ -523,8 +523,117 @@ implementation, as explained in [crypto.md].
 TBD
 
 # 12. Trusted Applications
-## Format
-Trusted Applications consists of a signed ELF file.
+
+## Static and Dynamic TAs
+
+There are two ways to implement Trusted Applications, static and dynamic.
```

Comment:
Maybe it's a rhetorical question, but I wonder why not to change the terminology and just use "OP-TEE os core service/module" instead of "Static/Pseudo TA".  It will help to avoid permanent confusion about Static TAs and writing documentation which explains the difference (frankly speaking, I've found four or five threads about the StaticTA/Normal TA difference for the last 2 months).